Pod/Classes/SKYPubsubContainer.h

```diff
@@ -19,11 +19,25 @@
 
 #import <Foundation/Foundation.h>
 
+@protocol SKYPubsubContainerDelegate <NSObject>
+
+@optional
+
+- (void)pubsubDidOpen;
```

Comment:
should be:

* `pubsubDidOpen:(SKYPubsubContainer *)container`
* `pubsubDidClose:(SKYPubsubContainer *)container`
* `pubsub:(SKYPubsubContainer *)container didFailWithError:(NSError *)error`